Resources/Textures/Clothing/Neck/Misc/pins.rsi/meta.json

```diff
@@ -1,74 +1,89 @@
 {
   "version": 1,
-  "license": "CC-BY-SA-3.0",
-  "copyright": "PixelTK leaves his mark on upstream",
+  "license": "CC-BY-SA-3.0, CC-BY-NC-4.0",
```

Comment:
I'm pretty sure you can only have one copyright, the test is failing because of it.

Reply:
How can I keep my own copyright in the same meta.json? Or do I have to make another RSI for it?

Reply:
Fixed, each pin has its own RSI now